Gemfile

```diff
@@ -5,8 +5,6 @@ gemspec
 group :development, :test do
   gem 'pry-byebug'
   gem 'rack-test'
-  gem 'rails'
-  gem 'rails-api'
```

Comment:
I didn't see these referenced anywhere so I removed them. It appears removing these two gems takes care of most of the reported vulnerabilities (they were never real vulnerabilities that could be exploited outside of the dev env).